tox.ini

```
@@ -53,9 +53,10 @@ commands =
     --out-dir {toxinidir}/dist/ \
     {toxinidir}
 
-[testenv:flake8]
-deps = flake8
-commands = flake8
+[testenv:lint]
+deps = pre-commit>=1.20.0
```

Let's keep it on a separate line, it's better for diffs.
```suggestion
deps =
    pre-commit>=1.20.0
```